gapic/src/main/com/google/gapid/views/PerformanceView.java

```diff
@@ -328,17 +329,25 @@ private void addColumnForMetric(Service.ProfilingData.GpuCounters.Metric metric)
       } else {
         Service.ProfilingData.GpuCounters.Perf perf = node.getPerfs().get(metric.getId());
         if (showEstimate) {
-          return perf.getEstimate() < 0 ? "" : unit.format(perf.getEstimate());
+          return perf.getEstimate() < 0 ? "" : format(perf.getEstimate(), unit, optimalOffset);
         } else {
-          String minStr = perf.getMin() < 0 ? "?" : unit.format(perf.getMin());
-          String maxStr = perf.getMax() < 0 ? "?" : unit.format(perf.getMax());
+          String minStr = perf.getMin() < 0 ? "?" : format(perf.getMin(), unit, optimalOffset);
+          String maxStr = perf.getMax() < 0 ? "?" : format(perf.getMax(), unit, optimalOffset);
           return minStr + " ~ " + maxStr;
         }
       }
     });
     column.getColumn().setAlignment(SWT.RIGHT);
   }
 
+  private String format(double value, Unit unit, int optimalOffset) {
+    if (unit.convertible() || optimalOffset >= 0) {
```

Comment:
In my understanding it needs a && operator here.

Reply:
Oops, thanks for catching it! Just fixed it with a new commit.

As for the term `offset`, it was chosen because it was already in quite some use in the unit file, for those `Time`, `Base2`, and `Base10` units. I think I will keep the term for now for consistence, but I modified the annotations a bit to make the term meaning more clear, thanks for suggestion!